wasmstation/src/renderer/proxy.rs

```diff
@@ -0,0 +1,66 @@
+use std::str::FromStr;
+
+use crate::Renderer;
+
+#[cfg(feature = "sdl2-renderer")]
+use super::Sdl2Renderer;
+#[cfg(feature = "wgpu-renderer")]
+use super::WgpuRenderer;
+
+pub enum ProxyRenderer {
+    #[cfg(feature = "wgpu-renderer")]
+    Wgpu(WgpuRenderer),
+    #[cfg(feature = "sdl2-renderer")]
+    Sdl2(Sdl2Renderer),
+}
+
+const AVAILABLE_RENDERERS: &[&str] = &[
+    #[cfg(feature = "wgpu-renderer")]
+    "wgpu",
+    #[cfg(feature = "sdl2-renderer")]
+    "sdl2",
+];
+
+impl ProxyRenderer {
+    pub fn available_renderers() -> &'static [&'static str] {
+        AVAILABLE_RENDERERS
+    }
+
+    pub fn set_display_scale(&mut self, scale: u32) {
+        // AFAIK, the wgpu renderer is the only one with display_scale at the moment.
```

Comment:
that's actually not true - there is a display scale as an inline constant '3' defined in Sdl2Renderer - but it doesn't have a field for that. 
We should do that in a later commit (no reason to change it in this PR)